apps/examples/testcase/le_tc/network/tc_net_ether.c

```diff
@@ -34,12 +34,12 @@
 #include "tc_internal.h"
 
 /**
-* @testcase		tc_net_ether_ntoa_p
-* @brief
-* @scenario
-* @apicovered		ether_ntoa()
-* @precondition
-* @postcondition
+* @testcase             : tc_net_ether_ntoa_p
```

Comment:
Don't mixing different kinds of change in one commit.
Please split changes to different commits.
1. refactoring (indentation, removing unnessary codes and so on)
2. adding new TCs
3. modifying net codes
4. so on